aspnetcore/signalr/configuration.md

```diff
@@ -0,0 +1,239 @@
+
+> [!NOTE]
+> In order to register Logging providers, you must install the necessary packages. See the [Built-in logging providers](xref:fundamentals/logging/index#built-in-logging-providers) section of the docs for a full list.
+
+For example, to enable Console logging, install the `Microsoft.Extensions.Logging.Console` NuGet package and call the `AddConsole` extension method:
+
+```csharp
+var connection = new HubConnectionBuilder()
+    .WithUrl("https://example.com/myhub")
+    .ConfigureLogging(logging => {
+        logging.SetMinimumLevel(LogLevel.Information);
+        logging.AddConsole();
+    })
+    .Build();
+```
+
+In the JavaScript client, a similar `configureLogging` method exists. Provide a `LogLevel` value indicating the minimum level of log messages to produce. Logs will be written to the browser console window.
+
+```javascript
+let connection = new signalR.HubConnectionBuilder()
+    .withUrl("/myhub")
+    .configureLogging(signalR.LogLevel.Information)
+```
+
+> [!NOTE]
+> To disable logging entirely, specify `signalR.LogLevel.None` in the `configureLogging` method.
+
+Log levels available to the JavaScript client are listed below. Setting the log level to one of these values will enable logging of messages at **or above** that level.
+
+| Level | Description |
+| ----- | ----------- |
+| `None` | No messages will be logged |
+| `Critical` | Messages that indicate a failure in the entire application |
+| `Error` | Messages that indicate a failure in the current operation |
+| `Warning` | Messages that indicate a non-fatal problem |
+| `Information` | Informational messages |
+| `Debug` | Diagnostic messages useful for debugging |
+| `Trace` | Very detailed diagnostic messages designed for diagnosing specific issues |
+
+### Configure allowed transports
+
+The transports used by SignalR can be configured in the `WithUrl` call (`withUrl` in JavaScript). A bitwise-OR of the values of `HttpTransportType` can be used to restrict the client to only use the specified transports. By default, all transports are enabled.
+
+For example, to disable the Server-Sent Events transport, but allow WebSockets and Long Polling connections:
+
+```csharp
+var connection = new HubConnectionBuilder()
+    .WithUrl("https://example.com/myhub", HttpTransportType.WebSockets | HttpTransportType.LongPolling)
+    .Build();
+```
+
+In the JavaScript client, transports are configured by setting the `transport` field on the options object provided to `withUrl`:
+
+```javascript
+let connection = new signalR.HubConnectionBuilder()
+    .withUrl("/myhub", { transport: signalR.HttpTransportType.WebSockets | signalR.HttpTransportType.LongPolling })
+    .build();
+```
+
+### Configure bearer authentication
```

Comment:
I think we need another article on this specifically with server side examples of consuming the token using the JWT auth handler

Reply:
I opened an issue for this.

Reply:
We have #5925 tracking that